templates/docs/examples/patterns/notifications/variants.html

```diff
@@ -1,216 +1,11 @@
 {% extends "_layouts/examples.html" %}
-{% block title %}Notifications / Variants {% endblock %}
-
-{% block standalone_css %}patterns_notifications{% endblock %}
```

Comment:
This line got lost in the migration. It is needed in the "Variants" and "Variants Dark" templates to link them to correct standalone stylesheet (in standalone examples).